dendrite-config.yaml

```diff
@@ -117,6 +117,13 @@ listen:
 tracing:
     # Config for the jaeger opentracing reporter.
     # See https://godoc.org/github.com/uber/jaeger-client-go/config#Configuration
-    # for documtation.
+    # for documentation.
     jaeger:
         disabled: true
+
+# The configuration for logs of dendrite
+logging:
+    # The logging level, must be one of debug, info, warn, error, fatal, panic.
+    level: "info"
+    # The file on which save logs. If commented, logs won't be saved by dendrite and only printed to stdout
+    # path: /var/log/dendrite.log
```

Comment:
If the expectation is for us to support multiple types of loggers, I'd be tempted to have an extensible format, e.g.:

```yaml
logging:
- type: file
  level: info
  params:
    path: /var/log/dendrite.log
```

This would be useful even in just the file case, where you might want to define a separate file for error logs (so that you could e.g. have different retention periods for the error logs)